man/oci-image-tool-unpack.1.md

```diff
@@ -20,6 +20,11 @@ oci-image-tool unpack \- Unpack an image or image source layout
 **--type**=""
   Type of the file to unpack. If unset, oci-image-tool will try to auto-detect the type. One of "imageLayout,image"
 
+**--platform**=""
+  Specify the os and architecture of the manifest, format is OS:Architecture.
+  e.g. --platform linux:amd64
```

Comment:
@stevvooe I have modified the introduction of information and added examples. PTAL